initial_website_build/initial_website_build.info.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Initial website build'
+description: 'This is the first build of the RWAHS site, it is not refined yet and still requires considerable cleaning up of theming and content, but all the basic config has been completed.'
+type: module
+core: 8.x
+dependencies:
+  - block
+  - block_content
+  - ckeditor
+  - comment
+  - datetime
+  - ds
+  - editor
+  - field
+  - file
+  - filter
+  - image
+  - jssor
```

Comment:
Nice slider - can't quite believe what you can do with it through builder, but I gather that's not enabled in the Drupal module.